### PR TITLE
Implemented HTML and CSS form

### PR DESCRIPTION
**Pull request by brenoxav**
We have implemented the following, according to the project requirements:

- Header
- Sign In Button
- Already Have an account message
- Forgot password message
- Logo and icons
- Form
- Footer

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,36 @@
...
+
+![screenshot](./assets/screenshot.png)
+
+## Built With
+
+- HTML & CSS
+
+## Live Demo
+
+[Live Demo Link](https://brenoxav.github.io/mint-clone-microverse/)
```
Comment (SpaYco) [inline README.md:15]: - [x] The Demo Link doesn't work!

File: index.html [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+                <div><a href="#">I forgot my User ID or Password</a></div>
+            </div>
+            <button type="button" class="signin-button"><ion-icon class="icon-svg" name="lock-closed-outline"></ion-icon>Sign In</button>
+        </div>
+        <div class="d-flex logos-wrapper">
+            
+            <img class="intuit-logo" src="assets/logo-intuit.png" alt="intuit logo">
+            <div>
+                <img class="logos" src="assets/header-icon.png" alt="header logos"/>
+            </div>
```
Comment (SpaYco) [inline index.html]: - [x]  It should show the Mint logo and text (logo on the left, text on the right) ([link to page](https://accounts.intuit.com/signup.html))
- [x]  It should show the Quickbooks logo and text (logo on the left, text on the right) ([link to page](https://accounts.intuit.com/signup.html))
- [x]  It should show the Turbotax logo and text (logo on the left, text on the right) ([link to page](https://accounts.intuit.com/signup.html))

**Review by SpaYco — CHANGES_REQUESTED** [review]
# Changes Requested 

Hello Team! 👋 
Awesome job so far!! 👍 you still need to make some changes though! 

## Required Changes ♻️ 

> Required changes are attached below this comment

## Optional

- [x] use the [`<nav>` tag](http://html5doctor.com/nav-element/) for the major navigation links

### please make the appropriate changes and ask for another code review ✔️ 

Happy Coding!

Best, 
Aziz Mejri 
_contact me on [twitter](https://twitter.com/iSpaYco) or slack (@SpaYco) if you need any help_

File: README.md (hunk 1 above)
Comment (brenoxav) [reply README.md:15]: The link should work now.

**Review by SpaYco — APPROVED** [review]
# Project Approved

thank you for making the required changes, your project is now approved!

### please don't forget to merge to master, good luck with the next projects!


Best, 
Aziz Mejri 
_contact me on [twitter](https://twitter.com/iSpaYco) or slack (@SpaYco) if you need any help_

**Pull request merged**